src/main/resources/jupyter/init-actions.sh

```diff
@@ -82,10 +83,15 @@ if [[ "${ROLE}" == 'Master' ]]; then
 
     docker-compose -f /etc/cluster-docker-compose.yaml up -d
 
+    # The Dataproc-provided spark-defaults.conf is compose'd into the Docker; append our custom settings to the end of it
+    docker exec -d ${JUPYTER_SERVER_NAME} /etc/construct-spark-defaults.sh
+
     # If a Jupyter extension was specified, poke it into the jupyter docker container.
     # Note: docker-compose doesn't appear to have the ability to execute a command after run, so we do this explicitly with docker exec commands.
     # See https://github.com/docker/compose/issues/1809
     if [ ! -z ${JUPYTER_EXTENSION_URI} ] ; then
       docker exec -d ${JUPYTER_SERVER_NAME} /etc/install-jupyter-extension.sh /etc/${JUPYTER_EXTENSION_ARCHIVE}
     fi
+
+    docker exec -d ${JUPYTER_SERVER_NAME} pyspark
```

Comment:
replaces `ENTRYPOINT=pyspark` in the Dockerfile since we want to tinker with the conf before launching pyspark